.github/workflows/update-openapi.yaml

```diff
@@ -40,22 +40,15 @@ jobs:
       - name: Copy openapi.json
         run: |
           cd registry
-          cp ./common/src/main/resources/META-INF/openapi.json ./app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json
-
-      - name: Make copy with Auth info
-        run: |
-          cd registry/utils/tools
-          mvn clean package
-          mvn exec:java -Dexec.mainClass="io.apicurio.registry.utils.tools.AddOpenApiAuth" -Dexec.args="../../app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json ../../app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi-auth.json"
-          mvn exec:java -Dexec.mainClass="io.apicurio.registry.utils.tools.TransformOpenApiForClientGen" -Dexec.args="../../app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json ../../app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi-gen.json"
+          cp ./common/src/main/resources/META-INF/openapi.json ./app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v3/openapi.json
```

Comment:
those changes should be redundant now, isn't it?

Reply:
We still do the copy operation because the canonical location of the `openapi.json` file is in META-INF (for java micro-profile reasons) but I also want it located in `api-specifications` so it can be shown in the browser when you navigate to the list of APIs at `http://localhost:8080/apis`.

But the -auth and -gen variants are now gone.

Reply:
would a sym-link work? `ln -s` ?